tectonic/msvc-include/unistd.h

```diff
@@ -0,0 +1,2 @@
+// For now, this is a total stub.
+// See https://stackoverflow.com/q/341817 for some things we could want.
```

Comment:
Since there is only 3 includes using `unistd.h`, it should be simpler and cleaner to just put an `#ifndef _MSVC` before such include?

I guess having something that provides the `HAVE_UNISTD_H` from gcc-rs would be great in the future.